.travis.yml

```diff
@@ -19,4 +21,15 @@ matrix:
       env: TEST_SUITE=lint
     - node_js: "4"
       env: TEST_SUITE=coveralls
-script: npm run $TEST_SUITE
+    - node_js: "4"
+      env: TEST_SUITE=saucelabs
+  allow_failures:
+    - env: TEST_SUITE=saucelabs
+fast_finish: true
+cache:
+  directories:
+    - node_modules
+before_script:
+  - npm install -g grunt-cli
+script:
+  - ./travis.sh
```

Comment:
I is real; i was thinking that you want this to see clear naming on travis.com outputs.

Let me know what you prefere and i will change accordingly! :)